doc/source/conf.py

```diff
@@ -7,6 +7,9 @@
 import time
 
 from ansys_sphinx_theme import (
+    ALL_NODES,
+    PARAGRAPHS,
+    TITLE,
```

Comment:
Minor suggestion. Since we are naming nodes in plural, let's do the same of this one:

```suggestion
    TITLES,
```